ksa_compliance/standard_doctypes/tax_category.py

```diff
@@ -0,0 +1,101 @@
+import frappe
+
+
+def tax_category_mapper(tax_category_id: str) -> dict:
+    zatca_category, custom_category_reason = frappe.get_value("Tax Category", {"name": tax_category_id},
+                                                              ["custom_zatca_category", "custom_category_reason"])
+    zatca_category = zatca_category if zatca_category else "Standard rate"
+    if zatca_category == "Standard rate":
+        return {
+            "category_code": category_to_code(zatca_category)
+        }
+    category, reason = zatca_category.split(" || ")
```

Comment:
Why are we splitting on `||`?

Reply:
the select is separated with ( || ) ex: Zero rated goods || Export of goods

Reply:
I don't understand. Who decided to put a `||` separated list of items in a data field instead of a child table?

Reply:
Re-reading your comment, you're saying it's a select? Why did we create it this way? Can you provide a link to the user story or reference document?